src/geometry/euclidean/monge_point.lean

```diff
@@ -544,14 +544,10 @@ end
 circumcenter in a side equals the circumradius, variant using a
 `finset`. -/
 lemma dist_orthocenter_reflection_circumcenter_finset (t : triangle ℝ P) {i₁ i₂ : fin 3}
-    (h : i₁ ≠ i₂) :
-  dist t.orthocenter (reflection (affine_span ℝ (t.points '' ↑({i₁, i₂} : finset (fin 3))))
-                                 t.circumcenter) =
+  (h : i₁ ≠ i₂) :
+  dist t.orthocenter (reflection (affine_span ℝ (t.points '' ({i₁, i₂}))) t.circumcenter) =
     t.circumradius :=
-begin
-  convert dist_orthocenter_reflection_circumcenter t h,
-  simp
-end
+dist_orthocenter_reflection_circumcenter t h
```

Comment:
This change is making this lemma into a duplicate of `dist_orthocenter_reflection_circumcenter`. (The idea of this lemma is for cases where you're using it with other lemmas such as `orthogonal_projection_circumcenter` where you naturally have a `finset` not a `set` of indices. But given the change of `orthogonal_projection_circumcenter` to be stated in terms of `face`, maybe lemmas should be stated with sets of indices and users should translate along the lines of what you have in `reflection_circumcenter_eq_affine_combination_of_points_with_circumcenter` as needed.)

Reply:
It would be nice to fix this by allowing the original (finset) version to be stated.  For this I need to add some nonemptiness instances for finsets, similar to the ones this PR is adding already for sets.  I'm running into a weird bug with this, will ask on Zulip.

Reply:
35959b3 adds another `nonempty` instance, (hopefully) allowing this lemma to return to its original statement.